app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java

```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
         //backBtn.setOnClickListener(new View.OnClickListener() {
         //@Override
         //public void onClick(View v) {
-        backBtn = (Button) findViewById(user_profile_back_button);
-        doneBtn = (Button) findViewById(R.id.button3);
-        childBtn = (Button) findViewById(R.id.button2);
-        firstnameEditText = (EditText) findViewById(R.id.editText2);
-        lastnameEditText = (EditText) findViewById(R.id.editText4);
-        ageEditText = (EditText) findViewById(R.id.editText5);
+        profilepic = (QuickContactBadge) findViewById(R.id.child_registration_child_portrait);
+        backBtn = (Button) findViewById(R.id.child_registration_back_button);
```

Comment:
any reasons you commented out methods and put everything right here?

Reply:
all comments on the page were either for code I was unsure of how to use or for something that I thought could be useful later on